packages/main/src/CardHeader.js

```diff
@@ -0,0 +1,310 @@
+		 *
+		 * @type {String}
+		 * @since 1.0.0-rc.15
+		 * @private
+		 * @defaultvalue ""
+		 */
+		ariaLabel: {
+			type: String,
+		},
+
+		/**
+		 * Receives id(or many ids) of the elements that label the component
+		 *
+		 * @type {String}
+		 * @defaultvalue ""
+		 * @private
+		 * @since 1.0.0-rc.15
+		 */
+		ariaLabelledby: {
+			type: String,
+			defaultValue: "",
+		},
+
+		_headerActive: {
+			type: Boolean,
+			noAttribute: true,
+		},
+	},
+	events: /** @lends sap.ui.webcomponents.main.CardHeader.prototype */ {
+
+		/**
+		 * Fired when the header is activated
+		 * by mouse/tap or by using the Enter or Space key.
+		 * <br><br>
+		 * <b>Note:</b> The event would be fired only if the <code>headerInteractive</code> property is set to true.
+		 * @event sap.ui.webcomponents.main.CarHeader#header-click
+		 * @public
+		 * @since 1.0.0-rc.15
+		 */
+		"header-click": {},
+	},
+};
+
+/**
+ * @class
+ * <h3 class="comment-api-title">Overview</h3>
+ *
+ * The <code>ui5-card-header</code> is a component that represents information in the header slot
+ * of the <code>ui5-card</code> component.
+ * The header can be used through several properties, such as: <code>titleText</code>, <code>subtitleText</code>, <code>status</code>
+ * and two slots: <code>avatar</code> and <code>action</code>.
+ *
+ * <h3>Keyboard handling</h3>
+ * In case you enable <code>headerInteractive</code> property, you can press the <code>ui5-card</code> header by Space and Enter keys.
+ *
+ * <h3>CSS Shadow Parts</h3>
+ *
+ * <ui5-link target="_blank" href="https://developer.mozilla.org/en-US/docs/Web/CSS/::part">CSS Shadow Parts</ui5-link> allow developers to style elements inside the Shadow DOM.
+ * <br>
+ * The <code>ui5-card</code> exposes the following CSS Shadow Parts:
+ * <ul>
+ * <li>title - Used to style the title of the card</li>
+ * <li>subtitle - Used to style the subtitle of the card</li>
+ * <li>status - Used to style the status of the card</li>
+ * </ul>
+ *
+ * <h3>ES6 Module Import</h3>
+ *
+ * <code>import "@ui5/webcomponents/dist/CardHeader";</code>
+ *
+ * @constructor
+ * @author SAP SE
+ * @alias sap.ui.webcomponents.main.CardHeader
+ * @extends sap.ui.webcomponents.base.UI5Element
+ * @tagname ui5-card-header
+ * @appenddocs CardHeader
+ * @public
+ */
+class CardHeader extends UI5Element {
+	constructor() {
+		super();
+
+		this.i18nBundle = getI18nBundle("@ui5/webcomponents");
+	}
+
+	static get metadata() {
+		return metadata;
+	}
+
+	static get render() {
+		return litRender;
+	}
+
+	static get template() {
+		return CardHeaderTemplate;
+	}
+
+	static get styles() {
+		return cardHeaderCss;
+	}
+
+	get classes() {
+		return {
+			"ui5-card-header": true,
+			"ui5-card-header--interactive": this.headerInteractive,
+			"ui5-card-header--active": this.headerInteractive && this._headerActive,
+		};
+	}
+
+	get icon() {
+		return !!this.avatar && this.avatar.startsWith("sap-icon://");
+	}
+
+	get image() {
+		return !!this.avatar && !this.icon;
```

Comment:
same

Reply:
do not think we need this fuction any more